src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+// Actions
+const CHECK = 'bookstore/books/CHECK';
+
+// Reducer
+export default function categoriesReducer(state = [], action = {}) {
+  switch (action.type) {
+    case CHECK:
+      return 'Under Construction';
+    default: return state;
```

Comment:
- [ ]Please make sure that this **action.type** for `under Construction` is rendered in the  category page not **check back.**

![Capture web_16-8-2022_14239_localhost](https://user-images.githubusercontent.com/61562626/184890468-25462fa0-a995-4797-9c0c-289c22f42d1a.jpeg)

Reply:
Ok, noted.